base/server/src/main/java/com/netscape/cmscore/usrgrp/User.java

```diff
@@ -102,6 +105,7 @@ public class User implements JSONSerializer {
     private String mUserType = null;
     private X509Certificate mx509Certs[] = null;
     private List<String> tpsProfiles = null;
+    private List<LDAPAttribute> mAttributes = null;
```

Comment:
The `m` prefix for field names is a legacy convention. For new code I don't think we should be using that anymore. Just use a normal Java naming convention.

Also, the `null` initialization is redundant since non-primitive fields are initialized to `null` by default.

Reply:
Yeah there was a lot here I didn't particularly like, I originally used a more modern convention but it looked ugly with two different styles so I kept it like the original code. I will leave this like it is so the diff for this change is minimised, then push a PR to clean this class up.